app/src/auth/components/AuthLayout/authLayout.pcss

```diff
@@ -1,73 +1,5 @@
 .html,
 .html body {
-  background: var(--blueDark6);
-}
-
-.root {
-  display: flex;
-  flex-direction: column;
-  font-size: 16px;
-  min-height: 100vh;
-  padding: 2em 0;
-}
-
-.outer {
-  display: flex;
-  flex-direction: column;
-  flex-grow: 1;
-  justify-content: space-between;
-}
-
-.content {
-  background: inherit;
-  display: flex;
-  flex-direction: column;
-  flex-grow: 0.8;
-  justify-content: center;
-  padding: 0 2em;
-}
-
-.inner {
-  display: flex;
-  flex-direction: column;
-  flex-grow: 0.1;
-  justify-content: space-between;
-}
-
-.logo {
-  padding-bottom: 3em;
-}
-
-.footer {
-  padding-left: 2em;
-  padding-right: 2em;
-}
-
-.panel {
-}
-
-@media (--md-up) {
-  .root {
-    justify-content: center;
-    padding: 3em 0;
-  }
-
-  .outer {
-    flex-grow: 0.2;
-    justify-content: start;
-  }
-
-  .inner {
-    flex-grow: 1;
-  }
-
-  .footer {
-    flex-grow: 1;
-    margin-bottom: 4em;
-  }
-
-  .content {
-    flex-grow: 1;
-    justify-content: initial;
-  }
+  background: #E5E5E5;
```

Comment:
This should be #F8F8F8. (I have done the same mistake when looking this from Figma. It quite often shows the default canvas background and not the selected page background color)